lib/private/do-with-connection.js

```diff
@@ -224,33 +226,37 @@ module.exports = function doWithConnection(options, done){
       // > is synchronous or not.  (This is not strictly necessary, it's just an optimization.)
       if (options.WET_MACHINES.releaseConnection.sync) {
         try {
-          options.WET_MACHINES.releaseConnection({ connection: db, meta: options.meta }).execSync();
+          options.WET_MACHINES.releaseConnection({
+            connection: db
```

Comment:
what about `meta`?  Realistically it probably doesn't matter whether it's there or not, but I think the safest thing would be not to change it in case it's relied upon